.dockerignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
This is what was *needed*, but you could also optimise the images by ignoring some more things. Like everything in .gitignore, `.gitignore` itself, `.git`